nextcloud-inotifyscan

```diff
@@ -188,8 +188,8 @@ def watch_instances(instances):
               user = None
               _logger.warning('cannot find valid user for %s', data_prefix)
         if user is not None:
-          path_prefix = '/'+user+'/files'+storage['mount_point']
-          data_prefix = storage['configuration']['datadir']
```

Comment:
#17 was caused by potential inconsistency here: `storage['mount_point']` ends with `/` but `storage['configuration']['datadir']` does not. Consistently removing trailing `/` should solve the problem.